helm/ph-ee-engine/ams-paygops/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ph-ee-connector-ams-paygops
+  namespace: paymenthub
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ph-ee-connector-ams-paygops
+subjects:
+- kind: ServiceAccount
+  name: ph-ee-connector-ams-paygops
```

Comment:
need to review this @avikganguly01, do we need to keep the service-account name and role name same as the service?

Reply:
Avik: Service account name should be PH Engine level I feel. I don't feel they need to be same as service unless it's for id account mapper kind of service.

Reply:
Let's keep the services, roles, and role binding at a granular level.